pkg/sql/parser/sql.y

```diff
@@ -1230,17 +1231,21 @@ preparable_stmt:
   {
     $$.val = $1.slct()
   }
-| create_stmt
-| drop_stmt
-| alter_table_stmt
 | insert_stmt
 | update_stmt
 | delete_stmt
+
+explainable_stmt:
+  preparable_stmt
+| create_stmt
+| drop_stmt
+| alter_table_stmt
```

Comment:
Postgres doesn't support DDLs in prepared statements, and I don't see a good reason for us to do so either. DDLs are not parameterizable and in general don't want to get repeated more than once.